wsdl/tp1-external.wsdl

```diff
@@ -215,7 +218,7 @@
   </wsdl:binding>
   <wsdl:service name="TP1ExternalServicesService">
     <wsdl:port binding="impl:TP1ExternalServicesSoapBinding" name="TP1ExternalServices">
-      <wsdlsoap:address location="http://localhost/1.0"/>
+      <wsdlsoap:address location="http://localhost/external/1.0"/>
```

Comment:
this is not related to the reporting, I just thought it should be visible from the url which service it is